app/public/index.html

```diff
@@ -49,6 +49,10 @@
   <link rel="icon" type="image/png" href="img/icons/favicon-96x96.png" />
 
   <style>
+    html {
+      position: unset !important;
```

Comment:
interesting. does it get overwritten somewhere?

Reply:
yes, strangely